### DISCUSSION TITLE
Version 2.60.0-preview.1

### DISCUSSION THREAD

**Discussion by yufeih** [dotnet/docfx#8305] [Announcement]
## .NET Core Support

This is the first preview of docfx that builds on top of .NET Core! 
Docfx is released as a [.NET Core Tool](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools) starting from `v2.60.0-preview.1`. .NET 6.x is required to run docfx. To install, run:

```bash
dotnet tool update -g docfx --version 2.60.0-preview.1
```

The rest of the command line experience hasn't changed and are backward compatible. We've also updated [docfx homepage](https://dotnet.github.io/docfx/) use the .NET Core version of docfx.

## Known Limitations
- Msbuild integration through [`docfx.console`](https://www.nuget.org/packages/docfx.console) is not supported in this preview.
- Plugins like [`memberpage`](https://www.nuget.org/packages/memberpage/), [`rest.tagpage`](https://www.nuget.org/packages/rest.tagpage), [`rest.operationpage`](https://www.nuget.org/packages/rest.operationpage) are not supported in this preview.
- Supports Windows only. Linux and MacOS are not supported in this preview.
- .NET Framework 4.x is still needed to build from source code in this preview.

### 🎉 New Features
* feat: Go net6.0 by @tibel in https://github.com/dotnet/docfx/pull/8238 https://github.com/dotnet/docfx/pull/8239 https://github.com/dotnet/docfx/pull/8241 https://github.com/dotnet/docfx/pull/8254
* feat: Upgrade to .NET Core by @yufeih in https://github.com/dotnet/docfx/pull/8279
* feat: Ship as global tool to NuGet.org by @yufeih in https://github.com/dotnet/docfx/pull/8280
* feat: Serve command using ASPNET.Core by @yufeih in https://github.com/dotnet/docfx/pull/8293
* feat: Deterministic output by @yufeih in https://github.com/dotnet/docfx/pull/8299
### 🐞 Bug Fixes
* fix: TOC word-break not being inserted in generic types by @julesabr in https://github.com/dotnet/docfx/pull/8217
* fix: Crash fixes for CopyInherited by @Sorpirit in https://github.com/dotnet/docfx/pull/7709
* fix: Fix on DocFX crashes on method parameter by @harriev9 in https://github.com/dotnet/docfx/pull/8208
### 🔧 Engineering
* build: Update package references by @tibel in https://github.com/dotnet/docfx/pull/8139
* build: Upgrade NuGet.Protocol by @YunshuMeg in https://github.com/dotnet/docfx/pull/8236
* build: Upgrade Newtonsoft.Json from 13.0.1 to 13.0.2 by @YunshuMeg in https://github.com/dotnet/docfx/pull/8257
* test: Add samples and snapshot testing by @yufeih in https://github.com/dotnet/docfx/pull/8299
### 📄 Documentation
* docs: Fix typo by @Lulalaby in https://github.com/dotnet/docfx/pull/8252
* docs: Update issue templates by @yufeih in https://github.com/dotnet/docfx/pull/8294

## New Contributors
* @julesabr made their first contribution in https://github.com/dotnet/docfx/pull/8217
* @Sorpirit made their first contribution in https://github.com/dotnet/docfx/pull/7709
* @harriev9 made their first contribution in https://github.com/dotnet/docfx/pull/8208
* @Lulalaby made their first contribution in https://github.com/dotnet/docfx/pull/8252

**Full Changelog**: https://github.com/dotnet/docfx/compare/v2.59.4...v2.60.0-preview.1

<hr /><em>This discussion was created from the release <a href='https://github.com/dotnet/docfx/releases/tag/v2.60.0-preview.1'>Version 2.60.0-preview.1</a>.</em>

**Comment by jtmueller** [comment]
"Supports Windows only. Linux and MacOS are not supported in this preview."

Ironically, this was the first version I successfully got working on MacOS.

**Reply by yufeih** [reply]
Glad that it works for you. I haven't tested everything on MacOS yet, expect it to officially support MacOS and linux in some future releases or prereleases.

**Reply by jtmueller** [reply]
I've not tried anything beyond the basic getting-started and generating from documentation comments, as today was my first time using docfx. The basic stuff at least seems to work better than the current release version does on M1 Macs, so well done! Looks great so far.
